File: app/src/main/java/com/fengniao/news/util/DateUtils.java
package com.fengniao.news.util;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Date;


public class DateUtils {


    /**
     * 日期转换为字符串
     *
     * @param format 日期格式  比如：yyyyMMdd
     * @param date   日期
     * @return
     */
    public static String dateToString(String format, Date date) {
        SimpleDateFormat mFormat = new SimpleDateFormat(format);
        return mFormat.format(date);
    }


}

File: app/src/main/java/com/fengniao/news/util/DateUtils.kt
package com.fengniao.news.util

import java.text.DateFormat
import java.text.SimpleDateFormat
import java.util.Date


object DateUtils {


    /**
     * 日期转换为字符串
     *
     * @param format 日期格式  比如：yyyyMMdd
     * @param date   日期
     * @return
     */
    fun dateToString(format: String, date: Date): String {
        val mFormat = SimpleDateFormat(format)
        return mFormat.format(date)
    }


}
